profiles/package.mask

```diff
@@ -1,3 +1,9 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-10)
+# this version of reflex is not currently compatible with ghc 8.10.x, but there
+# [is an pull request][1] open upstream to resolve this.
+# [1]: https://github.com/reflex-frp/reflex/pull/444
+=dev-haskell/reflex-0.8.0.0
```

Comment:
I like to at least go ahead and bring the new version in - this way, if someone else is working on a new package that needs the newer reflex, they'll see that it's already in the tree, as well as why it is masked (i.e. it can reduce duplication of effort)

I'll squash the mask into the commit that adds the package. I was reluctant to do so before due to trying to keep each commit "atomic"